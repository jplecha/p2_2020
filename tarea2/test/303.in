insertarAlFinal (6,1.8)
insertarAlFinal (6,7.6)
insertarAlFinal (7,3.7)
insertarAlFinal (3,9.4)
insertarAlFinal (3,3.9)
insertarAlFinal (6,1.9)
insertarAlFinal (7,7.9)
insertarAlFinal (5,8.1)
insertarAlFinal (7,3.7)
cambiarTodos
finalCadena
menorEnCadena
ordenar
esVaciaCadena
finalCadena
esLocalizador
finalCadena
finalCadena
finalCadena
kesimo 7
finalCadena
finalCadena
cambiarTodos
inicioCadena
