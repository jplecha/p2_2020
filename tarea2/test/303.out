==12682== Conditional jump or move depends on uninitialised value(s)
==12682==    at 0x4039E6: cambiarTodos(unsigned int, unsigned int, repCadena*) (usoTads.cpp:148)
==12682==    by 0x4022C4: main (principal.cpp:333)
==12682== 
principal: principal.cpp:288: int main(): Assertion `localizadorEnCadena(loc, cad)' failed.
1>Insertado al final.
2>Insertado al final.
3>Insertado al final.
4>Insertado al final.
5>Insertado al final.
6>Insertado al final.
7>Insertado al final.
8>Insertado al final.
9>Insertado al final.
10>Cambiados.
11>timeout: the monitored command dumped core
